docs/everest.md

```diff
@@ -0,0 +1,12 @@
+# Install Percona Distribution for PostgreSQL cluster using Everest
+
+[Percona Everest :octicons-link-external-16:](https://docs.percona.com/everest/) is an open source cloud-native database platform that helps developers deploy code faster, scale deployments rapidly, and reduce database administration overhead while regaining control over their data, database configuration, and DBaaS costs.
+
+It automates day-one and day-two database operations for Postgres, MySQL, and MongoDB databases within Kubernetes clusters.
+Percona Everest will provide you web GUI and allow launching a database instance with just a few clicks, as well as scale it, do routine maintenance tasks, such as software updates and patch management, backups, and monitoring.
+
+You can try it in action by [Installing Percona Everest :octicons-link-external-16:](https://docs.percona.com/everest/quickstart-guide/qs-overview.html) and [managing your first cluster :octicons-link-external-16:](https://docs.percona.com/everest/use/cluster-management.html).
+
+!!! note
+
+    Percona Everest is currently **a beta release** and it is **not recommended for production environments**. **As of today, we recommend using** [other Operator installation methods](System-Requirements.md#installation-guidelines), which are production-ready.
```

Comment:
Shall we mention it here? It is mentioned in Everest docs. Also I think we add one more place to update when Everest goes GA. Hard to remember.